test/ClientTest.cs

```diff
@@ -71,5 +73,37 @@ public void ActingAsTest()
 
             Assert.AreEqual(false, plainClientVars.MoveNext());
         }
+
+        [Test]
+        public void CreatePolicyTest()
+        {
+            string policyId = "vaultname/policyname";
+            string policyPath = $"test:///policies/{Client.GetAccountName()}/{Constants.KIND_POLICY}";
+            // we can check that sensitive data is not leaked (123456)
+            string policyResponseText = "apisecret:123456";
```

Comment:
What do you think about updating the response to look like the response we'd expect to receive from the Conjur API?  This struck me as confusing (especially the comment) as it's not what I'd expect a policy load response to look like.  Alternatively, we could verify the `Mocker` URL was called with `Client.Policy(policyId).LoadPolicy(ms)` rather than stub the response.